homeassistant/components/netatmo/camera.py

```diff
@@ -333,25 +303,25 @@ def _service_set_person_away(self, **kwargs):
                 if data.get("pseudo") == person:
                     person_id = pid
 
-        if person_id is not None:
-            self._data.set_persons_away(
+        if person_id:
```

Comment:
0 is not a valid ID?

Reply:
Yes, the person id is actually just a string in the form of `id:"6112f812-8b06-4127-90ab-dffsd0afb0f"`